vertx-mail-client/src/test/java/io/vertx/ext/mail/MailAttachmentTest.java

```diff
@@ -35,12 +37,14 @@ public void testConstructor() {
   @Test
   public void testToJson() {
     assertEquals("{}", new MailAttachment().toJson().encode());
-    assertEquals("{\"data\":\"ZGF0YQ==\",\"contentType\":\"text/plain\",\"disposition\":\"inline\",\"description\":\"description\"}",
+    assertEquals("{\"data\":\"ZGF0YQ==\",\"contentType\":\"text/plain\",\"disposition\":\"inline\",\"description\":\"description\",\"contentId\":\"id1@domain.net\",\"headers\":{}}",
```

Comment:
No <> around the contentId. Is it normal (not homogeneous with the documentation)